Is this package supposed to be compatible with python3?

```
$ nix build --no-link --keep-going --max-jobs 4 --option build-use-sandbox true -f /home/joerg/.cache/nix-review/pr-57013/build.nix
error: build of '/nix/store/6bqgikqaxnbmsf9h5prnw44zsqq3awaf-python3.7-marisa-1.3.40.drv' on 'ssh://nix@eve.thalheim.io' failed: builder for '/nix/store/6bqgikqaxnbmsf9h5prnw44zsqq3awaf-python3.7-marisa-1.3.40.drv' failed with exit code 1
builder for '/nix/store/6bqgikqaxnbmsf9h5prnw44zsqq3awaf-python3.7-marisa-1.3.40.drv' failed with exit code 1; last 10 log lines:
  marisa-swig_wrap.cxx:2549:13: note: suggested alternative: ‘PyLong_FromVoidPtr’
         obj = PyCObject_FromVoidPtr(descriptor, NULL);
               ^~~~~~~~~~~~~~~~~~~~~
               PyLong_FromVoidPtr
  marisa-swig_wrap.cxx: At global scope:
  marisa-swig_wrap.cxx:2512:22: warning: ‘swig_empty_runtime_method_table’ defined but not used [-Wunused-variable]
     static PyMethodDef swig_empty_runtime_method_table[] = { {NULL, NULL, 0, NULL} };/* Sentinel */
                        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  error: command 'gcc' failed with exit status 1
  builder for '/nix/store/6bqgikqaxnbmsf9h5prnw44zsqq3awaf-python3.7-marisa-1.3.40.drv' failed with exit code 1
cannot build derivation '/nix/store/rb736bdh5v642fv8x9zq4f0f4fqb11q8-env.drv': 1 dependencies couldn't be built
[5 built (1 failed), 18 copied (120.4 MiB), 31.1 MiB DL]
error: build of '/nix/store/rb736bdh5v642fv8x9zq4f0f4fqb11q8-env.drv' failed
https://github.com/NixOS/nixpkgs/pull/57013
1 package failed to build:
python37Packages.marisa
```

Otherwise please use the `disable` attribute to mark it as broken on python3.

--- a/pkgs/development/python-modules/marisa/default.nix
+++ b/pkgs/development/python-modules/marisa/default.nix
@@ -1,18 +1,22 @@
-{ stdenv, buildPythonPackage, fetchFromGitHub, marisa, swig }:
+{ stdenv, buildPythonPackage, fetchFromGitHub, marisa, swig
+, isPy3k
+}:
 
 buildPythonPackage rec {
   pname = "marisa";
   version = "1.3.40";
 
+  disabled = isPy3k;
+
   src = fetchFromGitHub {
     owner = "s-yata";
     repo  = "marisa-trie";
-    rev   = "59e410597981475bae94d9d9eb252c1d9790dc2f";
-    sha256 = "0z4bf55np08q3cbi6gvj3cpw3zp8kf2d0jq6k74pjk066m7rapbb";
+    rev   = "8dba9850b89d7828ebf33b8ab84df2b54d31260b";
+    sha256 = "0pkp9fggk53lxlicfwrskgx33qplc4v6njbavlnz4x4z63zd4933";
   };
 
   nativeBuildInputs = [ swig marisa ];
-  buildinputs = [ marisa ];
+  buildInputs = [ marisa ];
 
   sourceRoot = "${src.name}/bindings/python";
 

